web/src/routes/admin/repair/+page.svelte

```diff
@@ -118,7 +118,10 @@
     try {
       const matched = await loadAndMatch([filename]);
       if (matched) {
-        notificationController.show({ message: `Matched 1 item`, type: NotificationType.Info });
+        notificationController.show({
+          message: $t('admin.matched_items', { values: { count: 1 } }),
```

Comment:
Maybe prefix that with `repair_`? Could also think about prefixing everything here with `repair_` (where it makes sense). In the admin settings we also tried to prefix stuff that belongs to one section